python_scripts/ensemble_random_forest.py

```diff
@@ -149,19 +152,28 @@
 # By default, `RandomForestRegressor` disables feature subsampling while
 # `RandomForestClassifier` uses `max_features=np.sqrt(n_features)`. These
 # default values reflect good practices given in the scientific literature.
-# However, `max_features` is one of the hyperparameter to consider when tuning
-# a random forests.
 #
-# Note that bagging ensemble exposes a parameter `max_features` as well.
-# However, the process is different since the base estimator is not necessarily
-# a decision tree. Indeed, the features subsampling take place at the model
-# level by selecting a subset of feature on each bootstrap sample.
+# However, `max_features` is one of the hyperparameters to consider when tuning
+# a random forests:
+# - too much randomness in the trees can lead to underfitting base models and
```

Comment:
underfitting -> underfitted

Reply:
For the record Wikipedia says "underfitted" which I believed was not correct English but so be it.